datasets/alt/alt.py

```diff
@@ -45,68 +45,76 @@
+        datasets.BuilderConfig(name="alt-jp", version=datasets.Version("1.0.0"), description="Japanese ALT"),
+        datasets.BuilderConfig(name="alt-my", version=datasets.Version("1.0.0"), description="Myanmar ALT"),
+        datasets.BuilderConfig(name="alt-km", version=datasets.Version("1.0.0"), description="Khmer ALT"),
+        datasets.BuilderConfig(
+            name="alt-my-transliteration",
+            version=datasets.Version("1.0.0"),
+            description="Myanmar-English Transliteration Dataset",
+        ),
         datasets.BuilderConfig(
-            name="alt", version=datasets.Version("1.0.0"), description="Asian Language Treebank Parallel Corpus"
+            name="alt-my-west-transliteration",
+            version=datasets.Version("1.0.0"),
+            description="Latin-Myanmar Transliteration Dataset",
         ),
-        # datasets.BuilderConfig(
-        #     name="alt-en", version=datasets.Version("1.0.0"), description="English ALT"
-        # ),
-        # datasets.BuilderConfig(
-        #     name="alt-jp", version=datasets.Version("1.0.0"), description="Japanese ALT"
-        # ),
-        # datasets.BuilderConfig(
-        #     name="alt-my", version=datasets.Version("1.0.0"), description="Myanmar ALT"
-        # ),
-        # datasets.BuilderConfig(
-        #     name="alt-km", version=datasets.Version("1.0.0"), description="Khmer ALT"
-        # ),
-        # datasets.BuilderConfig(
-        #     name="alt-my-transliteration", version=datasets.Version("1.0.0"), description="Myanmar-English Transliteration Data Set"
-        # ),
-        # datasets.BuilderConfig(
-        #     name="alt-my-west-transliteration", version=datasets.Version("1.0.0"), description="Latin-Myanmar Transliteration Dataset"
-        # )
     ]
 
     DEFAULT_CONFIG_NAME = "alt"
 
     def _info(self):
-        if self.config.name == "alt":
+        if self.config.name.startswith("alt-parallel"):
             features = datasets.Features(
                 {
                     "SNT.URLID": datasets.Value("string"),
                     "SNT.URLID.SNTID": datasets.Value("string"),
                     "url": datasets.Value("string"),
-                    "bg": datasets.Value("string"),
-                    "en": datasets.Value("string"),
-                    "en_tok": datasets.Value("string"),
-                    "fil": datasets.Value("string"),
-                    "hi": datasets.Value("string"),
-                    "id": datasets.Value("string"),
-                    "ja": datasets.Value("string"),
-                    "khm": datasets.Value("string"),
-                    "lo": datasets.Value("string"),
-                    "ms": datasets.Value("string"),
-                    "my": datasets.Value("string"),
-                    "th": datasets.Value("string"),
-                    "vi": datasets.Value("string"),
-                    "zh": datasets.Value("string"),
```

Comment:
Well actually the `Translation` feature type allow to have many languages, not only 2 !
```python
datasets.Translation(languages=[...]) the languages list can have many datasets, not only 2
```
So you can keep the `alt` configuration you had with all the languages.

Could you re-add this config but using the `Translation` feature type instead ?

Reply:
sorry if that wasn't clear ^^'